src/components/Person/index.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import SPerson from './style';
+
+function Person({ name, firstname, city, url }) {
+  return (
+    <SPerson>
+      <div>
+        <li>
+          <h3>
+            {name} {firstname}
```

Comment:
Sry but it's `lastname`now and not name..
I marked all the change made in the group and sent the new database